frontend/app/routes/$lang+/_protected+/alerts+/unsubscribe+/success.tsx

```diff
@@ -28,45 +33,53 @@ export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
 });
 
 export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
+  const log = getLogger('alerts.unsubscribe.success');
+
+  const auditService = getAuditService();
+  const instrumentationService = getInstrumentationService();
   const raoidcService = await getRaoidcService();
+  const subscriptionService = getSubscriptionService();
 
   await raoidcService.handleSessionValidation(request, session);
 
-  const log = getLogger('alerts.unsubscribe.success');
   const t = await getFixedT(request, handle.i18nNamespaces);
-
   const meta = { title: t('gcweb:meta.title.template', { title: t('alerts:success.page-title') }) };
 
   const userInfoToken: UserinfoToken = session.get('userInfoToken');
-  if (!userInfoToken.sin) {
-    log.warn('SIN was not found in session, returning a 500 status');
-    throw new Response(null, { status: 500 });
-  }
-  const alertSubscription = await getSubscriptionService().getSubscription(userInfoToken.sin);
+  invariant(userInfoToken.sin, 'Expected userInfoToken.sin to be defined');
+  const alertSubscription = await subscriptionService.getSubscription(userInfoToken.sin);
 
   if (alertSubscription?.subscribed) {
     log.warn('User is already subscribed, returning a 400 status');
+    instrumentationService.countHttpStatus('alerts.unsubscribe-success', 400);
     throw new Response(null, { status: 400 });
   }
```

Comment:
Not really a part of your PR, but wouldn't it make sense to return a 2xx response here? I imagine this as a PUT request, which is idempotent (not sure if that's what the API requires, but it *should* be a put).

Reply:
Also I just noticed this code is a part of `unsubscribe+/succes.tsx` ... is there a typo here? Seems like the check wouldn't make sense in the context of unsubscribing.

Reply:
> Also I just noticed this code is a part of `unsubscribe+/succes.tsx` ... is there a typo here? Seems like the check wouldn't make sense in the context of unsubscribing.

I agree. I think that this check was put in place to prevent users from accessing the `unsubscribe/success` page without filling out the previous page's form to confirm unsubscribing, but I'm not certain (tagging @nibivi77 for any comments). If that is the case, I feel we should redirect instead.

I have updated this PR to remove this check completely for now. We can add flow state for the alerts pages in future PR.